test/e2e/commands/node_pfx_kill.test.mjs

```diff
@@ -1,25 +0,0 @@
-/**
- * Copyright (C) 2024 Hedera Hashgraph, LLC
- *
- * Licensed under the Apache License, Version 2.0 (the ""License"");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an ""AS IS"" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * @jest-environment steps
- */
-
-import { describe } from '@jest/globals'
-import { constants } from '../../../src/core/index.mjs'
-import { e2eNodeKeyRefreshTest } from '../e2e_node_util.js'
-
-describe('NodeCommand', () => {
-  e2eNodeKeyRefreshTest(constants.KEY_FORMAT_PFX, 'node-cmd-e2e-pfx', 'kill')
```

Comment:
I was using PEM to test stop + refresh, then PFX to test kill + refresh.

So, if we remove this test, we need to test the kill + refresh scenario somewhere.

Reply:
Refactor the test to use PEM